lib/rules/image-alt.json

```
@@ -8,7 +8,9 @@
     "wcag111",
     "section508",
     "section508.22.a",
-    "ACT"
+    "ACT",
+    "TTv5",
+    "TT7.a"
```

It's sort of both A and B I think, since decorative images need an empty alt.

```suggestion
    "TTv5",
    "TT7.a",
    "TT7.b"
```